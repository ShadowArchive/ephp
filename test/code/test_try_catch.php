<?php

try {
  print "hey!\n";
  throw new Exception("it's broken");
  print "no way!\n";
} catch (Exception $e) {
  print "Exception: " . $e->getMessage() . "\n";
}

