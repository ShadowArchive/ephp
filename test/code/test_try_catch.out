hey!
Exception: it's broken
